src/runtime/capabilities-new.ts

```diff
@@ -263,127 +296,195 @@ export abstract class BooleanCapability extends Capability {
 }
 
 export class Queryable extends BooleanCapability {
-  constructor(isQueryable: boolean) { super(isQueryable); }
-
-  static fromAnnotations(annotations: AnnotationRef[]): Capability {
+  static fromAnnotations(annotations: AnnotationRef[] = []): Capability {
     const queryableAnnotations =
         annotations.filter(annotation => annotation.name === 'queryable');
+    if (queryableAnnotations.length === 0) {
+      return null;
+    }
     assert(queryableAnnotations.length <= 1, `Containing multiple queryable annotations`);
```

Comment:
This assertion should probably be `=== 1` now

Reply:
done, thanks.